pkg/provider/cloud/kubevirt/namespace.go

```diff
@@ -67,3 +67,13 @@ func deleteNamespace(ctx context.Context, name string, client ctrlruntimeclient.
 
 	return client.Delete(ctx, ns)
 }
+
+func reconcileKubeVirtDiskCacheNamespace(ctx context.Context, name string, client ctrlruntimeclient.Client) error {
+	creators := []reconciling.NamedNamespaceCreatorGetter{
```

Comment:
It's missing to add the `FinalizerClonerRoleBinding` in the cluster (example line 43)

Reply:
Its added in `reconcileDataVolumeClonerRoleRoleBinding()` in data_volume.go

Reply:
my bad, right.